modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v1_0_0/UpgradeDynamicDataMapping.java

```diff
@@ -2353,9 +2358,13 @@ protected void addAssetEntry(
 			assetEntry.setHeight(height);
 			assetEntry.setWidth(width);
 			assetEntry.setPriority(priority);
-			assetEntry.setViewCount(viewCount);
 
 			_assetEntryLocalService.updateAssetEntry(assetEntry);
+
+			ViewCountServiceUtil.incrementViewCount(
+				companyId,
+				_classNameLocalService.getClassNameId(AssetEntry.class),
```

Comment:
Maybe just add methods with the Class<?> instead of only classNameId. That way we don't have to refactor as much of the usages.